src/modules/addToUI.js

```diff
@@ -0,0 +1,63 @@
+import ToDo from './todo.js';
+// Add items to UI
+const addToList = () => {
+  const todoList = document.getElementById('todo-list');
+  todoList.innerHTML = '';
+
+  ToDo.list.forEach((item) => {
+    const listItem = document.createElement('li');
+    listItem.setAttribute('id', item.index);
+    listItem.classList = 'item-container';
+
+    listItem.innerHTML = `
+    <input class="checkbox" type="checkbox">
+    <input type="text" class="input" value='${item.description}'>
+    <button class="cancel-btn"><i class="fa fa-trash" aria-hidden="true"></i></button>
+    `;
+
+    // <p class="text-area">${item.description}</p>
+
+    todoList.appendChild(listItem);
+
+    const checkbox = listItem.querySelector('input');
```

Comment:
Do not commit old pieces of code as inline comments. They will make your project look messy.
and also it's better to clean the white lines.